src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -131,6 +166,54 @@ public void verifyTopLeaderboard() {
     verifyAdVisibleInSlot(presentLeaderboardSelector, presentLeaderboard);
   }
 
+  public void verifyFandomDesktopArticleTopLeaderboard() {
```

Comment:
Moved. I used "AdsFandomObject" name because that is convention of other objects ("AdsFliteObject", "AdsKruxObject")